modules/priors/tests/Rcheck_reference.log

```diff
@@ -69,14 +69,7 @@ The Date field is over a month old.
 * checking package directory ... OK
 * checking for future file timestamps ... OK
 * checking DESCRIPTION meta-information ... NOTE
-Author field differs from that derived from Authors@R
-  Author:    ‘David LeBauer’
-  Authors@R: ‘David LeBauer [aut, cre], University of Illinois, NCSA [cph]’
-
-Maintainer field differs from that derived from Authors@R
-  Maintainer: ‘David LeBauer <dlebauer@illinois.edu>’
-  Authors@R:  ‘David LeBauer <dlebauer@email.arizona.edu>’
-
+License stub is invalid DCF.
```

Comment:
Will these go away when we switch licenses?

Reply:
Yes. I'll update this PR to remove them from the packages we've finished relicensing.